doc/releases/changelog-0.27.0.md

```diff
@@ -236,139 +397,133 @@
+  [(#3041)](https://github.com/PennyLaneAI/pennylane/pull/3041)
+  [(#3090)](https://github.com/PennyLaneAI/pennylane/pull/3090)
+  [(#3069)](https://github.com/PennyLaneAI/pennylane/pull/3069)
+  [(#3137)](https://github.com/PennyLaneAI/pennylane/pull/3137)
+  [(#3127)](https://github.com/PennyLaneAI/pennylane/pull/3127)
+  [(#3099)](https://github.com/PennyLaneAI/pennylane/pull/3099)
+  [(#3098)](https://github.com/PennyLaneAI/pennylane/pull/3098)
+  [(#3095)](https://github.com/PennyLaneAI/pennylane/pull/3095)
+  [(#3091)](https://github.com/PennyLaneAI/pennylane/pull/3091)
+  [(#3176)](https://github.com/PennyLaneAI/pennylane/pull/3176)
+  [(#3170)](https://github.com/PennyLaneAI/pennylane/pull/3170)
+  [(#3194)](https://github.com/PennyLaneAI/pennylane/pull/3194)
+  [(#3267)](https://github.com/PennyLaneAI/pennylane/pull/3267)
+  [(#3234)](https://github.com/PennyLaneAI/pennylane/pull/3234)
+  [(#3232)](https://github.com/PennyLaneAI/pennylane/pull/3232)
+  [(#3223)](https://github.com/PennyLaneAI/pennylane/pull/3223)
+  [(#3222)](https://github.com/PennyLaneAI/pennylane/pull/3222)
+
+  In v0.26, we introduced `qml.enable_return()`, which separates measurements into their own tensors. The motivation of this change is the deprecation of ragged `ndarray` creation in NumPy.
+
+  With this release, we're continuing to elevate this feature by adding support for gradient transforms and some interfaces.
+
+  ```python
+  import torch 
+
+  qml.enable_return()
+  dev = qml.device("default.qubit", wires=2)
+
+  @qml.qnode(dev, interface="torch")
+  def circuit(x):
+      qml.Hadamard(0)
+      qml.Hadamard(1)
+      qml.CRX(x, wires=[0, 1])
+      return qml.probs(wires=[0, 1]), qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))
+  ```
 
   ```pycon
-  >>> dataset.write("./path/to/dataset.dat")
-  >>> read_dataset = qml.data.Dataset()
-  >>> read_dataset.read("./path/to/dataset.dat")
-  >>> read_dataset.data_name
-  'Example'
-  >>> read_dataset.hamiltonian
-  (0.5) [X1]
-  + (1) [Z0]
-  >>> read_dataset.energies
-  array([-1.5, -0.5,  0.5,  1.5])
+  >>> circuit(torch.pi/3)
+  (tensor([0.2500, 0.2500, 0.2500, 0.2500], dtype=torch.float64),
+   tensor(0., dtype=torch.float64))
   ```
 
+  For more details, please
+  [refer to the documentation of `qml.enable_return()`](https://docs.pennylane.ai/en/stable/code/api/pennylane.enable_return.html?highlight=enable_return#pennylane.enable_return).
+
 <h3>Improvements</h3>
 
+* The `Exp` class decomposes into a `PauliRot` class if the coefficient is imaginary and the base operator is a Pauli Word.
+  [(#3249)](https://github.com/PennyLaneAI/pennylane/pull/3249)
+
 * Added a `samples_computational_basis` attribute to the `MeasurementProcess` and `QuantumScript` classes to track if computational basis samples are being generated when `qml.sample` or `qml.counts` are called without specifying an observable.
   [(#3207)](https://github.com/PennyLaneAI/pennylane/pull/3207)
```

Comment:
This one is listed twice, once here and once in bug fixes

Reply:
The PR with tag #3181 is also listed twice, once in improvements and once in bug fixes